db.go

```diff
@@ -134,6 +163,11 @@ type TxListReply struct {
 	Err     error
 }
 
+// BlockAddrIndex represents the indexing structure for addresses.
+// It maps a hash160 to a list of transaction locations within a block that
+// either pays to or spends from the passed UTXO for the hash160.
+type BlockAddrIndex map[[ripemd160.Size]byte][]*btcwire.TxLoc
```

Comment:
I think it would be better to export the ripemd160.Size as a AddrIndexKeySize constant in this package.  That way callers don't have to import the ripemd160 package just to get a constant.

Something like:
```Go
// AddrIndexKeySize is the number of bytes used by keys into the BlockAddrIndex.
const AddrIndexKeySize = ripemd160.Size

// BlockAddrIndex ...
type BlockAddrIndex [[AddrIndexKeySize]byte][]*tcwire.TxLoc
```